Source/Scene/Scene.js

```diff
@@ -1219,17 +1365,15 @@ define([
             }
         }
 
-        scene._globeDepth.update(context);
-        scene._globeDepth.clear(context, passState, clearColor);
-
+        // If supported, configure OIT to use the globe depth frame buffer and clear the OIT frame buffer.
```

Comment:
Shame on me for pointing it out, but throughout "frame buffer" should be one word.  This is why our code is `Framebuffer`, not `FrameBuffer`.